.github/dependabot.yml

```diff
@@ -10,3 +10,23 @@ updates:
     open-pull-requests-limit: 10
     schedule:
       interval: "daily"
+  - package-ecosystem: "pip"
+    directory: "/tools/accuracy"
+    open-pull-requests-limit: 10
```

Comment:
open-pull-request liminit should be dropped.

Updating other requirements.txt is not in the scope of this PR.